### PR TITLE
[1.6.x] Update trusty images

### PR DESCRIPTION
**Pull request by r00ta**
depends on https://github.com/kiegroup/kogito-examples/pull/662 first

Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

**WARNING! Please make sure you are opening your PR against `master` branch!**

- [ ] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [ ] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Pull Request</b>  
  Please add comment: <b>Jenkins retest this</b>
 
* <b>Quarkus LTS checks</b>  
  Please add comment: <b>Jenkins run LTS</b>

* <b>Native checks</b>  
  Please add comment: <b>Jenkins run native</b>

</details>

### CODE REVIEW

File: trusty-demonstration/kubernetes/README.md [hunk 1]
```diff
@@ -47,41 +47,45 @@ minikube addons enable ingress
...
 
 ```bash 
 PROJECT_NAME=trusty-demo
 kubectl create namespace "$PROJECT_NAME"
 ```
 
+Set the Kogito release version
+
+```bash
+KOGITO_VERSION=v1.6.0
```
Comment (manstis) [inline trusty-demonstration/kubernetes/README.md:60]: The [old](https://github.com/kiegroup/kogito-examples/pull/663/files#diff-5f8e9190a622e4580fc34d0a1315953517a95d2c93671f0e1425bd315185e780L53) value was just `1.4.0`. Now you've added "v" i.e. `v1.6.0`. Is that correct?

**Comment by r00ta** [comment]
Jenkins retest this please

File: trusty-demonstration/kubernetes/README.md (hunk 1 above)
Comment (r00ta) [reply trusty-demonstration/kubernetes/README.md:60]: Hi @manstis , yes, that's correct :) see the artifacts links here https://github.com/kiegroup/kogito-operator/releases/tag/v1.5.0

**Comment by r00ta** [comment]
Jenkins retest this please

File: trusty-demonstration/kubernetes/README.md (hunk 1 above)
Comment (manstis) [reply trusty-demonstration/kubernetes/README.md:60]: OK.. buuuut `1.4.0` was also tagged `v1.4.0` and the download artifacts had `v` in their URLs too:

https://github.com/kiegroup/kogito-operator/releases/tag/v1.4.0

So why do you now include the `v` in the version?

Sorry. Am I being thick?

Comment (radtriste) [reply trusty-demonstration/kubernetes/README.md:60]: the `v` is just for operator... other repos are tagged without the `v`

**Review by manstis — APPROVED** [review]

**Comment by manstis** [comment]
@radtriste Can you please review and approve too?

I'm no expert here so think somebody more knowledgeable should also review and approve.

**Review by radtriste — APPROVED** [review]
lgtm.

I guess it would be great to have a script to automate that part and which would be called also in the `Jenkinsfile.deploy` file to setup this version correctly on version set. wdyt @r00ta ?

**Comment by r00ta** [comment]
> lgtm.
> 
> I guess it would be great to have a script to automate that part and which would be called also in the `Jenkinsfile.deploy` file to setup this version correctly on version set. wdyt @r00ta ?

Hi @radtriste , yes I think we already discussed and agreed it in the past. But we did not put effort on this yet.. I just created https://issues.redhat.com/browse/KOGITO-5041 fyi

**Pull request merged**